python/taichi/misc/util.py

```diff
@@ -230,26 +197,6 @@ def get_kernel_stats():
     return _ti_core.get_kernel_stats()
 
 
-def print_async_stats(include_kernel_profiler=False):
```

Comment:
This function seems related to our async project. @k-ye wdyt?

Reply:
@strongoier I deleted mainly because it wasn't even in the `__all__` in this file, so it's very likely that this one is not used...

Reply:
This is some random util function...